unstructured/documents/elements.py

```diff
@@ -143,6 +143,9 @@ class ElementMetadata:
     sent_to: Optional[List[str]] = None
     subject: Optional[str] = None
 
+    # Epub specific metadata fields
```

Comment:
Can we describe this as `# Document section fields` since other document types could have sections as well?